### PR TITLE
Homework Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by natata08**
I did all the tasks in one file - homework.js.

### CODE REVIEW

File: javascript/javascript1/week1/homework.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+// Step 2
+// My freecodecamp username - @Natata08
+
+//Step 3
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1986; //number
+const yearFuture = 2025; //number
+const age = yearFuture - yearOfBirth;
+const messageAboutFututeAge = 'You will be ' + age + ' years old in ' + yearFuture;
```
Comment (Rassibassi) [inline javascript/javascript1/week1/homework.js]: `Futute` is a typo, it seems you would like to say `Future`

File: javascript/javascript1/week1/homework.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = false; //boolean
+let messageAboutDogAge;
+
+if (shouldShowResultInDogYears) {
+  messageAboutDogAge = 'Your dog will be ' + dogYear * 7 + ' dog years old in ' + dogYearFuture;
+} else {
+  messageAboutDogAge = 'Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture;
+}
```
Comment (Rassibassi) [inline javascript/javascript1/week1/homework.js]: Very nice. In line 23 and line 25 you are repeating yourself quite a lot. Most of these two lines are the same. Can you rewrite this, such that only the `dogYear` is changed based on the condition, and the string concatenation is done outside of the if-else clause.


See https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

File: javascript/javascript1/week1/homework.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  houseHeightInM,
+  houseDepthInM,
+  gardenSizeInM2,
+  announcedHousePrice
+) {
+  const volumeInMeters = houseWidthInM * houseHeightInM * houseDepthInM;
+  const housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+  let message;
+  if (housePrice < announcedHousePrice) {
+    message = name + ' is paying too much for this house. The real price is: ' + housePrice;
```
Comment (Rassibassi) [inline javascript/javascript1/week1/homework.js]: You could make nice use of string template literals here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Example:

```javascript
const weather = "sunny"
const degrees = 25
const outputString = `The weather is ${weather} at ${degrees} degrees.`
console.log(outputString)
```

**Review by Rassibassi — COMMENTED** [review]
Good job I can see you have been using prettier!

File: javascript/javascript1/week1/homework.js (hunk 1 above)
Comment (jason-vasilev) [reply javascript/javascript1/week1/homework.js]: suggestion: You could install [Code Spell Checker for VS Code](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker). It spellchecks your variable names, comments, and everything else. Misspelled words are shown with a squiggly blue underline.

File: javascript/javascript1/week1/homework.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  'Concepts',
+  'Insights',
+  'Ventures',
+  'Networks',
+  'Analytics',
+];
+
+const firstWord = firstWords[Math.floor(Math.random() * 10)];
+const secondWord = secondWords[Math.floor(Math.random() * 10)];
+const charactersAmount = firstWord.length + secondWord.length;
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/homework.js]: You've done great naming the variables in the other exercises, but here there could be a bit of improvement. Those are not just `firstWord` and `secondWord`. They are randomly selected. `randomFirstWord` and `randomSecondWord` might be better.
`charactersLength` could a bit better than `charactersAmount`.

**Review by jason-vasilev — APPROVED** [review]
Nothing critical. You've done very well, also doing the optional last exercise. If you'd like, take the time to rename some of the variables, replace some of the [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation) with [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

File: javascript/javascript1/week1/homework.js (hunk 1 above)
Comment (natata08) [reply javascript/javascript1/week1/homework.js]: I fixed it. Thank you so much. Code Spell Checker helped me find another typo.

File: javascript/javascript1/week1/homework.js (hunk 2 above)
Comment (natata08) [reply javascript/javascript1/week1/homework.js]: Done! It really good concept, thank you. 
I hope I fixed it correctly. I've found out about a ternary operator and replaced all if-else with it.

File: javascript/javascript1/week1/homework.js (hunk 3 above)
Comment (natata08) [reply javascript/javascript1/week1/homework.js]: It's amazingly more convenient! Thanks, I've implemented it everywhere.

File: javascript/javascript1/week1/homework.js (hunk 4 above)
Comment (natata08) [reply javascript/javascript1/week1/homework.js]: Thanks for the feedback! It's very useful!
I hope I've fixed everything.

**Comment by natata08** [comment]
Thank you very much for your feedback!

**Pull request merged**